src/lib/util/cdrom.cpp

```diff
@@ -2506,6 +2587,7 @@ std::error_condition cdrom_file::parse_cue(std::string_view tocfname, toc &outto
 
 bool cdrom_file::is_gdicue(std::string_view tocfname)
 {
+	static char token[512];
```

Comment:
Why is this static?  It prevents the function from being called reentrantly.  I can’t immediately see a reason for it to outlive an invocation.

Reply:
`parse_cue` and `parse_toc` both handle the token buffer used by the tokenizer in that way and I remade `is_gdicue` in the style of those existing functions, so it ended up like that. As for a general why it's like that in cdrom.cpp, I don't know but probably something to do with the general lack of modernization in the file.